site/public/js/forum.js

```diff
@@ -966,6 +974,7 @@ function readThreadStatusValues() {
 
 function dynamicScrollLoadPage(element, atEnd) {
     const load_page = $(element).data(atEnd?'next_page':'prev_page');
+    // eslint-disable-next-line eqeqeq
     if (load_page == 0) {
```

Comment:
The same behavior was produced when scrolling through the discussion forum posts after replacing `load_page == 0` with `load_page === 0`.  When getting the type of `load_page`, `number` is returned.